worlds/smw/Options.py

```diff
@@ -255,25 +332,33 @@ class MarioPalette(Choice):
     default = 0
 
 
-class ForegroundPaletteShuffle(Toggle):
-    """
-    Whether to shuffle level foreground palettes
-    """
-    display_name = "Foreground Palette Shuffle"
-
-
-class BackgroundPaletteShuffle(Toggle):
+class LevelPaletteShuffle(Choice):
     """
-    Whether to shuffle level background palettes
+    Whether to shuffle level palettes
+    off: Do not shuffle palettes
+    on_legacy: Uses only the palette sets from the original game
+    on_curated: Uses custom palette sets created by some people
```

Comment:
This snake case format for option values is inconsistent with at least `SFXShuffle`, which capitalizes the option as it would be on the webhost.

Reply:
> created by some people
This is also vague in a sort of silly way, and could maybe be rephrased.